### PR TITLE
drop file functionality in CoreFileInput.vue component to the UI

### PR DESCRIPTION
**Pull request by SurajRaika**
image

### CODE REVIEW

File: ui/src/core_components/input/CoreFileInput.vue [hunk 1]
```diff
@@ -105,30 +133,37 @@ const encodeFile = async (file: File) => {
 
 const fileChange = async (ev: InputEvent) => {
 	const el = ev.target as HTMLInputElement;
-	if (!el.files || el.files.length == 0) return;
+	handlefiles(el.files);
+};
+
+const handlefiles = async (Files: FileList) => {
+	if (!Files || Files.length == 0) return;
+	window.alert("handlefiles");
```
Comment (FabienArcellier) [inline ui/src/core_components/input/CoreFileInput.vue:141]: I got this alert message when I uploaded the file. Is this a development artifact that remained in the code?

File: ui/src/core_components/input/CoreFileInput.vue [hunk 2]
```diff
@@ -105,30 +133,37 @@ const encodeFile = async (file: File) => {
 
 const fileChange = async (ev: InputEvent) => {
 	const el = ev.target as HTMLInputElement;
-	if (!el.files || el.files.length == 0) return;
+	handlefiles(el.files);
+};
+
+const handlefiles = async (Files: FileList) => {
```
Comment (FabienArcellier) [inline ui/src/core_components/input/CoreFileInput.vue:139]: All function arguments appear to be written in lowercase. Generaly, these are the types that start with a capital letter. Is there a constraint to write this one with a capital letter? Would it improve the readability of the code to use minuscule ?

**Comment by FabienArcellier** [comment]
User experience feels great :100: . Is it intentional that you can't rename the component ? If it is, you should remove the label box.

image

**Comment by ramedina86** [comment]
@FabienArcellier once we're done addressing the remaining bugs, can we polish this? 

Let me know how you'd go about it, perhaps new branch, merge against that branch, do the changes, submit against dev? I can edit the PR and change branch if needed

**Pull request closed without merging**